kit/inputs/src/UIMaterialTextView/useMaterialTextViewChildren.tsx

```diff
@@ -84,10 +84,11 @@ export function useMaterialTextViewChildren(
     isFocused: boolean,
     isHovered: boolean,
     clear: () => void,
+    success?: boolean,
 ) {
     const clearButton = useClearButton(inputHasValue, isFocused, isHovered, clear);
 
-    if (clearButton) {
+    if (clearButton && !success) {
```

Comment:
Please leave a comment with a case where this is needed, because it doesn't look like general case and probably we should change it in a future to sth more generic, I just don't know right how it should be

Reply:
@savelichalex done

Reply:
How about renaming the parameter? Into something like `hideClearButton`? As it doesn't bring any context like success (which might be different)